DigitalOffice/Tests/CheckRightsServiceTests/Controllers/CheckRightsControllerTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
```

Comment:
Remove.

Reply:
Well. Why?

Reply:
Because we agreed to remove controller tests

Reply:
fixed